src/Tribe/Views/V2/Messages.php

```diff
@@ -162,14 +170,22 @@ static function ( array &$value, $message_type ) {
 					$updated_messages,
 					static function ( array &$value, $message_type ) {
 						ksort( $value );
-						// Keep only the last message.
-						$highest = reset( $value );
-						$value   = [ reset( $highest ) ];
+						// Keep the highest priority (lower number).
+						$highest = (array) reset( $value );
+						// Keep only the first message.
+						$value = [ reset( $highest ) ];
 					}
 				);
 				break;
 			default:
-				break;
+			case static::RENDER_STRATEGY_LIST:
+			array_walk(
+				$updated_messages,
+				static function ( array &$value, $message_type ) {
+					ksort( $value );
+				}
+			);
+			break;
```

Comment:
:no_entry_sign: Terminating statement must be indented to the same level as the CASE body

<sub>hash: 8b67451f21dcea74fa7936ed8b96008f</sub>